x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4

```diff
@@ -106,11 +106,12 @@ field
     ;
 
 fromCommand
-    : FROM indexIdentifier (COMMA indexIdentifier)* metadata?
+    : FROM indexString (COMMA indexString)* metadata?
```

Comment:
I chose indexString instead of indexSource since we use source (to declare source commands).

Reply:
`indexPattern` would be a common alternative.